crates/cli/src/forge/cmd/script/build.rs

```diff
@@ -167,8 +167,10 @@ impl ScriptArgs {
                     }
                 }
 
-                let tc: ContractBytecode = contract.into();
-                highlevel_known_contracts.insert(id, tc.unwrap());
+                if let Ok(tc) = ContractBytecode::from(contract).try_into() {
```

Comment:
why is this a `try_into` now?

Reply:
`tc.unwrap()` panics if `bytecode` or `deployed_bytecode` is `None`, which is the case for `interfaces`. So the code checks whether the contract is deployable with `try_into()`. As far as I understand it `highlevel_known_contracts` should only contain deployable contracts.